### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by umairidris**
Some of the docs were a bit out of date which was confusing for me when getting the app up and running. I am fixing them here. Please suggest any improvements/reverts.

Additional comments:
- david-dm.org is down (not sure if forever) so the dependency statuses don't load
- I am not sure if the rest of the doc is up to date (i.e. is the design link still used/maintained?)
- Who is "me" when referring to who to ask for access to db/design? It should be an email or some other source of contact

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33109

**Comment by umairidris** [comment]
Not sure what checks are failing... I don't have to travis.

**Comment by sabeurthabti** [comment]
@umairidris nothing you did, I will go ahead and approve this. 

@ahmedre, anythign we can do about this?

```bash
+ docker run --name github_pr567 -p 8000 -d quran/quran.com-frontend:github_pr567
docker: Error response from daemon: Conflict. The name "/github_pr567" is already in use by container 
```

**Comment by naveed-ahmad** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33112

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33114

File: README.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
+- Get the source by running `git clone https://github.com/quran/quran.com-frontend/` or creating a [fork]
+- Run `npm install` to do first time installation of all dependencies
+- Run `npm run dev` to start the dev server
+- Open `http://localhost:8000` in your browser to see the app.
 
 ## Backend
-Current at: https://github.com/quran/quran-api-rails
+The API source is at https://github.com/quran/quran-api-rails
+
 DB is private, message me for access.
```
Comment (mmahalwy) [inline README.md]: me is me :)

File: README.md [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
 
+The dev server uses the production API by default. If you want to use a local API server, follow the instructions in the API repo and run the server locally then update the API_URL field in app.json to point to the local address.
```
Comment (mmahalwy) [inline README.md]: uses staging api

**Comment by mmahalwy** [comment]
ty ty!

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33115

**Comment by sabeurthabti** [comment]
#570 caused a conflict

**Comment by ahmedre** [comment]
not sure what why (it properly stopped and removed the container before starting the new one), but the rebuild did fix it. we can keep an eye on it insha'Allah.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33126

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33127

**Comment by umairidris** [comment]
Seems like there was a bug open for some of the changes in this doc: #531, however it now discusses some Windows related problems so I don't know whether we can close it or not. Might be worth renaming the title at the very least.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33128

**Comment by mmahalwy** [comment]
@umairidris let's open a new one

**Pull request merged**